packages/js-dpp/test/integration/dataContract/validation/validateDataContractFactory.spec.js

```diff
@@ -1734,6 +1734,34 @@ describe('validateDataContractFactory', function main() {
     });
   });
 
```

Comment:
We need to add a test for `InvalidIdentityPublicKeySecurityLevelError`

Reply:
Added